src/msgpack_asgi/_middleware.py

```diff
@@ -1,25 +1,40 @@
 import json
+from typing import Any, Callable
 
 import msgpack
 from starlette.datastructures import Headers, MutableHeaders
 from starlette.types import ASGIApp, Message, Receive, Scope, Send
 
 
 class MessagePackMiddleware:
-    def __init__(self, app: ASGIApp) -> None:
+    def __init__(
+        self,
+        app: ASGIApp,
```

Comment:
(Nit) Seems like a good use case for making these keyword-only?

```suggestion
        app: ASGIApp,
        *,
```